.github/workflows/drivers-skipped-checks.yml

```diff
@@ -0,0 +1,92 @@
+# Required checks with path filtering rules will block pull requests from merging if they change only the excluded files.
+# This is a workaround to allow the PR to be merged.
+# https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+name: Driver Tests
+
+on:
+  pull_request:
+    paths:
+      - "docs/**"
+      - "**.md"
+      - "frontend/test/**"
+      - "enterprise/frontend/test/**"
+
+jobs:
+
+  be-tests-druid-ee:
+    runs-on: ubuntu-20.04
+    steps:
+      - run: |
+          echo "Didn't run due to conditional filtering"
```

Comment:
6 months from now I might wonder what the conditional filtering is. Should this message indicate that it is skipped because no driver/backend code was touched?

Reply:
Since we have a bunch of these `skipped-checks` workflows, I'm thinking of a follow-up PR that adds such a message/reminder in all the said workflows at once.